modules/persistent-storage-csi-smb-cifs-create-pv.adoc

```diff
@@ -0,0 +1,192 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-create-pv_{context}"]
+= Creating PVs/PVCs with SMB shares
+
+To create a persistent volume (PV) and persistent volume claim (PVC) to consume Server Message Block protocol (SMB) shares:
+
+.Prerequisites
+* Access to the {product-title} web console.
+* {FeatureName} CSI Driver Operator and driver installed.
+* Storage class for CIFS/SMB already created (see _Creating a storage class for CIFS/SMB_).
+
+.Procedure
+
+. Create a PV:
++
+Run the following command with the following example YAML file:
++
+[source,cli]
+----
+oc create -f <pv-file-name>.yaml <1>
+----
+<1> The name of the PV YAML file.
++
+.Example PV YAML file
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  annotations:
+    pv.kubernetes.io/provisioned-by: smb.csi.k8s.io
+  name: <pv-name> <1>
+spec:
+  capacity:
+    storage: 100Gi
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: <sc-name> <2>
+  mountOptions:
+    - dir_mode=0777
+    - file_mode=0777
+  csi:
+    driver: smb.csi.k8s.io
+    volumeHandle: smb-server.default.svc.cluster.local/share## <3>
+    volumeAttributes:
+      source: //<hostname>/<shares> <4>
+    nodeStageSecretRef:
+      name: <secret-name> <5>
+      namespace: <namespace> <6>
+----
+<1> The name of the PV.
+<2> The name of the CIFS/SMB storage class that you created earlier (see _Prerequisites_).
+<3> `volumeHandle` format: {smb-server-address}#{sub-dir-name}#{share-name}. Ensure that this value is unique for every share in the cluster.
```

Comment:
I feel like even if the format does not exactly contain text you'd find in a file, it maybe should be put in backticks since it kind of represents text that would end up in the YAML. What do you think?
```suggestion
<3> `volumeHandle` format: `{smb-server-address}#{sub-dir-name}#{share-name}`. Ensure that this value is unique for every share in the cluster.
```